clippy_utils/src/ast_utils.rs

```diff
@@ -571,3 +572,32 @@ pub fn eq_mac_args(l: &MacArgs, r: &MacArgs) -> bool {
         _ => false,
     }
 }
+
+/// Extract args from an assert-like macro.
+/// Currently working with:
+/// - `assert!`, `assert_eq!` and `assert_ne!`
+/// - `debug_assert!`, `debug_assert_eq!` and `debug_assert_ne!`
+/// For example:
+/// `assert!(expr)` will return Some([expr])
```

Comment:
This would not work for `assert!` anymore?

Reply:
There is no reason it would not work for `assert!`, or did you mean something else?

Reply:
Ah yes you're right! Hum... I'll update the function so it's not always returnins two arguments.

Reply:
Now it's returning a `vec`, not great but at least it works with `assert` too.